CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ if(POLICY CMP0051)
   cmake_policy(SET CMP0051 NEW)
 endif()
 
+if(POLICY CMP0054)
```

Comment:
I used the version 3.3.2 for cmake which uses 'NEW' policy by default.
Without the policty setting, this cmake version prints many warning messages for almost `if()`  even if the default is 'NEW'.
So I should set the policy to 'OLD' or 'NEW'.

If I set to 'NEW', it will break other platforms which used cmake version 2.x.